### PR TITLE
Swcd 2457 credits block component

### PR DESCRIPTION
**Pull request by mattb-hippo**

### CODE REVIEW

File: Childrens-Social-Care-CPD/Models/ContextModel.cs [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 namespace Childrens_Social_Care_CPD.Models;
 
+public record PublishDates (
+    DateTime ?FirstPublishedAt,
```
Comment (nealhippo) [inline Childrens-Social-Care-CPD/Models/ContextModel.cs:6]: Going to assume that only the Date is used from this date DataTime as UTC does not seem. to handled?

**Review by nealhippo — APPROVED** [review]
There is a comment on PublishDates wont stop approval (picky).

File: Childrens-Social-Care-CPD/Models/ContextModel.cs (hunk 1 above)
Comment (mattb-hippo) [reply Childrens-Social-Care-CPD/Models/ContextModel.cs:6]: It is only the date that gets displayed to the user.

**Pull request merged**